pythia/scripts/features/extract_features_vmb.py

```diff
@@ -0,0 +1,179 @@
+# Requires maskrcnn-benchmark to be built and installed
```

Comment:
Should this be `vqa-maskrcnn-benchmark`? We can be more explicit here so that it doesn't create any confusion. Maybe add a link to the gitlab repo url as well in comments.

Reply:
This is mostly for our own internal use. `vqa-maskrcnn-benchmark` is not properly public yet.